mir_perception/mir_object_recognition/ros/src/multimodal_object_recognition_node.cpp

```diff
@@ -755,9 +853,10 @@ void MultimodalObjectRecognitionROS::adjustObjectPose(mas_perception_msgs::Objec
         ROS_WARN_STREAM("Updated container height: " << object_list.objects[i].pose.pose.position.z );
       }
     }
+    */
     
     // Update axis or bolt pose
-    if (object_list.objects[i].name == "M20_100" || object_list.objects[i].name == "AXIS")
+    if (object_list.objects[i].name == "M20_100" || object_list.objects[i].name == "AXIS" || object_list.objects[i].name == "SCREWDRIVER")
```

Comment:
for this PR, it's fine. But in general we should remove all object names from the code, and move them to the object info config file, and only perform checks based on object info.